dev/PushNotifications/PushNotificationChannel.h

```diff
@@ -3,9 +3,7 @@
 
 #pragma once
 #include "Microsoft.Windows.PushNotifications.PushNotificationChannel.g.h"
-
 #include "winrt/Windows.Networking.PushNotifications.h"
-#include <TerminalVelocityFeatures-PushNotifications.h>
```

Comment:
Should we remove this header from the project?

Reply:
Depends.

If you can think of another feature coming soon that's going to need this header, it might be nice to avoid the churn of deleting and then re-adding it later.

On the other hand, if there's nothing on the horizon, then get rid of it so we don't accumulate cruft.

Reply:
It's a generated file and although it isn't included anywhere at the moment, it will become handy next time we want to add a few experimental APIs.

Removing the xml and the generation code is beyond the minimum work required for this fix.